e2e/test/DeviceStreamingE2ETests.cs

```diff
@@ -289,7 +289,7 @@ private async Task TestDeviceStreamingAsync(TestDeviceType type, ITransportSetti
 
                 Assert.IsNotNull(clientRequest, "Received an unexpected null device streaming request");
 
-                _log.WriteLine("Device streaming request received (name=" + clientRequest.Name + "; url=" + clientRequest.Url + "; authToken=" + clientRequest.AuthorizationToken + ")");
+                _log.WriteLine("Device streaming request received (name=" + clientRequest.Name + "; url=" + clientRequest.Uri + "; authToken=" + clientRequest.AuthorizationToken + ")");
```

Comment:
string interpolation?

Reply:
I agree, but I'm trying to limit how much I clean up in this branch, rather than doing it in master.

This change was just from a rename, so I prefer to leave the rest of the file alone.

Reply:
On second thought, this is a device-streaming specific file, so I'll clean it up.